wallet/src/api/routes.rs

```diff
@@ -1,12 +1,32 @@
 use actix::prelude::*;
+use failure::Fail;
 use futures::{future, Future};
-use jsonrpc_core::{Middleware, Params};
+use jsonrpc_core::{self as rpc, Middleware, Params};
 use jsonrpc_pubsub::{PubSubHandler, PubSubMetadata};
-use serde_json::{json, to_value};
+use serde_json::{self as json, json, to_value};
```

Comment:
Please avoid ambiguous module renames.
```suggestion
use serde_json::{json, to_value};
```

Reply:
Let's discuss this in another channel

Reply:
Ok, you won :laughing: